vexination/walker/Cargo.toml

```diff
@@ -4,19 +4,20 @@ version = "0.1.0"
 edition = "2021"
 
 [dependencies]
-serde = { version = "1.0", features = ["derive"] }
-serde_json = "1.0.68"
-tokio = { version = "1.0", features = ["full"] }
-log = "0.4"
-reqwest = { version = "0.11", features = ["json", "stream"] }
-trustification-infrastructure = { path = "../../infrastructure" }
-trustification-auth = { path = "../../auth" }
-clap = { version = "4", features = ["derive"] }
 anyhow = "1"
-csaf = "0.5.0"
+async-trait = "0.1"
+bytes = "1"
+clap = "4.3.3"
+humantime = "2.1.0"
+log = "0.4"
 csaf-walker = { version = "=0.6.0-alpha.6", default-features = false, features = ["crypto-openssl"] }
-walker-common = { version = "=0.6.0-alpha.6", features = [] }
-url = { version = "2.3.1", features = ["serde"] }
+serde_json = "1.0.68"
 time = "0.3.21"
-humantime = "2.1.0"
-prometheus = "0.13.3"
+tokio = {version = "1.28.2", features = ["full"] }
+tracing = "0.1"
+url = "2.4.0"
+walker-common = "=0.6.0-alpha.6"
+walker-extras = "=0.6.0-alpha.6"
+
+trustification-auth = { path = "../../auth" }
+trustification-infrastructure = { path = "../../infrastructure" }
```

Comment:
Did you reorder these? Why? Makes it very difficult to tell what changed.

Reply:
Old school Java rules were "alphabetical". This seems alphabeatized.

Reply:
It's rust-fmt automatically formatting when saving file.

Reply:
Well, turn that off, please. Or, if it's important to you, include a "whitespace-only" commit before making the essential changes.

Reply:
Ok, it's not `rustfmt` (which doesn't care about toml files, but I have an extension "Better TOML Language support" that does the formatting.

The alphabetic ordering makes sense, unless there is another way to organize packages ?